apps/trouble-host-unavailable.html.markerb

```diff
@@ -0,0 +1,83 @@
+---
+title: Troubleshoot apps when a host is unavailable
+objective: 
+layout: docs
+nav: firecracker
+---
+
+Fly.io has multiple hosts (physical hardware) in every supported region. A host could be unavailable because of a hardware failure, connectivity issues, or other reasons. A host issue might result in a complete or intermittent outage, or generally degraded performance for apps and databases without redundancy. A host could be down for an hour or a day, or sometimes longer. We understand that this can be frustrating if your app doesn’t have extra Machines or replicated databases for redundancy.
+
+For production apps, or any apps that require high availability, we strongly recommend running multiple machines and volumes for redundancy in the case of a single host failure. This goes double for apps with volumes, as volumes are pinned to specific hosts.
+
+<div class="important icon">
+**Important:** The steps for recovery in this document don’t guarantee that you’ll be able to access your app if you have a single Machine&mdash;or a single Machine with an attached volume&mdash;on an a host that’s affected by an ongoing issue.
+</div>
+
+## Notification of host issues
+
+You’ll get a notification on the Status page in your [dashboard](https://fly.io/dashboard/) when there’s a host issue that affects one or more of your apps. We don’t post individual host issues to our main status page because a host issue usually affects a small number of users in a single region.
+
+## For apps with one Machine and no volumes
+
+You can usually restore an app with one Machine and no volumes by scaling down and back up, or by deploying a new version of the app.
+
+### Scale the app to get new Machines
+
+For apps without volumes, you can bring up a new Machine by scaling to zero and then back up again. The new Machines will be placed on a healthy host.
+
+1. Scale the app to zero Machines:
+
+    ```cmd
+    fly scale count 0
+    ```
+
+2. Scale back up to 2 or more Machines:
+
+    ```cmd
+    fly scale count 2
+    ```
+
+<div class="note icon">
+**Note:** When you scale all your app's Machines to zero, the Machine size resets to shared-cpu-1x with 256MB of RAM. Use one of the `--vm-` options to size the Machines. For example: `fly scale count 2 --vm-size shared-cpu-4x`. See the [fly scale count docs](docs/flyctl/scale-count/) for options.
+</div>
+
+
+### Deploy the app to get new Machines
+
+In most cases deploying a new version of your app with `fly deploy` should also work for an app without volumes, as the new machines will be deployed to healthy hosts.
+
+<div class="important icon">
+<b>Important:</b> Running a new deploy during a host outage generally won't destroy the existing Machine. When the impacted host returns, the Machine that was on it may return as well. In the case of a deploy during an outage, this can result in one Machine running an older version of your code. In that case you should destroy the outdated Machine if it comes back.
```

Comment:
I don't know about anyone else but "deploy" sticks in my craw as a noun. That's a Heroku-ism, isn't it?